feat(docs): add planned integrations for ENSDb snapshots, CLI, and webhooks (#2107)
namehash/ensnode: docs/ensnode.io/src/content/docs/docs/integrate/integration-options/ensdb.mdx

**ENSDb** is an open standard that stores the live onchain **ENSv2** state in a carefully-crafted data model within a PostgreSQL database. Because it's plain Postgres, you can use *any* language with a Postgres driver — **TypeScript**, **Python**, **Rust**, **Go**, and more.

> :::tip[Coming soon: snapshots & CLI]

Reviewer: ```suggestion
:::tip[Coming soon: ensdb-cli & ENSDb snapshots]
```

> We're building [**ENSDb snapshots & an `ensdb-cli`**](/docs/integrate/ensdb-snapshots-and-cli) so you can pull down a fresh ENSDb in minutes instead of paying for a full historical RPC backfill, diff snapshots across versions to validate indexing changes, and operate ENSNode like a database. **Planned - not shipping yet**, but worth peeking at if your roadmap depends on a fast or cheap ENSDb bootstrap.

Reviewer: ```suggestion
We're building [**`ensdb-cli` & ENSDb snapshots**](/docs/integrate/ensdb-cli) so you can pull down a fresh ENSDb in minutes instead of paying for a full historical RPC backfill among many other benefits.
```
